Source/WebKit/NetworkProcess/Notifications/Cocoa/WebPushDaemonConnectionCocoa.mm

```diff
@@ -54,7 +54,22 @@ static void addVersionAndEncodedMessageToDictionary(Vector<uint8_t>&& message, x
 
     Daemon::Encoder encoder;
     encoder.encode(m_configuration);
-    Daemon::Connection::send(dictionaryFromMessage(WebPushD::MessageType::UpdateConnectionConfiguration, encoder.takeBuffer()).get());
+    Daemon::Connection::sendWithReply(dictionaryFromMessage(WebPushD::MessageType::UpdateConnectionConfiguration, encoder.takeBuffer()).get(), [](xpc_object_t reply) {
+        size_t dataSize = 0;
+        const void* data = nullptr;
+        if (xpc_get_type(reply) == XPC_TYPE_DICTIONARY)
+            data = xpc_dictionary_get_data(reply, protocolEncodedMessageKey, &dataSize);
+
+        if (!dataSize) {
+            RELEASE_LOG(Push, "Established connection to webpushd with no protocol version");
+            return;
+        }
+
+        Daemon::Decoder decoder({ static_cast<const uint8_t*>(data), dataSize });
+        std::optional<uint64_t> maybeVersion;
+        decoder >> maybeVersion;
+        RELEASE_LOG(Push, "Established connection to webpushd with protocol version: %llu", maybeVersion ? *maybeVersion : 0);
```

Comment:
`maybeVersion.value_or(0)`

Reply:
@beidson says he prefers to not return the protocol version so I removed all of this logic from the latest PR.